pkg/new_storage/cos.go

```diff
@@ -78,30 +78,48 @@ func (c *COS) DeleteFile(key string) error {
 }
 
 func (c *COS) Walk(cosPath string, recursive bool, process func(RemoteFile) error) error {
-	prefix := path.Join(c.Config.Path, cosPath)
+	// COS needs prefix ended with "/".
+	prefix := path.Join(c.Config.Path, cosPath) + "/"
```

Comment:
looks like you have wrong config
where `path` in `cos` section is empty
better will check this case or add config validation

Reply:
> looks like you have wrong config where `path` in `cos` section is empty better will check this case or add config validation

I use COS the first time, the path config is empty, It does not work. When check this API doc, u can find the "/" suffixed with every path in their query case. (%2F = "/", the html code).

Reply:
> looks like you have wrong config where `path` in `cos` section is empty better will check this case or add config validation

I used COS the first time, the path config is exactly empty, It did not work either. When check this API doc, I can find the "/" suffixed with every path in their query case. (%2F = "/", the html code).